### DISCUSSION TITLE
Help: how to simulate drag-motion like browser while zooming

### DISCUSSION THREAD

**Discussion by meotimdihia** [pmndrs/use-gesture#573] [Q&A]
This is my example while zooming an image by double tapping or pinching:
https://codesandbox.io/s/billowing-star-9wk7y3

I copied from your example (gesture-card-zoom): https://codesandbox.io/s/github/pmndrs/use-gesture/tree/main/demo/src/sandboxes/card-zoom

But motion while zooming by dragging is very slow. Please help me how to do it so smoothly + fast like when you zoom a website and drag it around, for example: go to https://google.com on your phone, zoom it and drag.

## our example:
https://user-images.githubusercontent.com/300961/211195470-a7153cef-74e0-42a3-9e27-350c1baeea4b.mov

## This is how the browser works:
https://user-images.githubusercontent.com/300961/211195359-5e26e100-f768-427b-bce9-d905004a862e.mov

**Comment by meotimdihia** [comment]
@dbismut Sorry about tagging you. But do you know how to fix this problem?

**Reply by dbismut** [reply]
You'll need to use inertia or decay in your gesture handler for your animation. This is possible with some animation libs such as react-spring or framer-motion. It's beyond the scope of this lib and I would advise against trying to replicate native scroll container, you'll bang your head against the walls.

**Reply by meotimdihia** [reply]
@dbismut  I am using `react-spring` too. This is what my code looks like: 

https://codesandbox.io/s/billowing-star-9wk7y3

If it is easy please help me to update the code. Or not, let's leave here.

**Reply by dbismut** [reply]
It's definitely not easy I'm sorry.

**Reply by meotimdihia** [reply]
Thanks for answering me. 😄
